src/index.html

```diff
@@ -4,9 +4,30 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>WebPack Exercise</title>
+  <title>To-Do List App</title>
 </head>
 <body>
-  <h1>Hello webpack!</h1>
+  <div class="container">
+    <main>
```

Comment:
These tags in your code are not in use, what do you think about removing these tags for creating a more readable code.

Reply:
I will remove the div with class=container since as you mentioned I am not using it for styling. Nevertheless, the HTML best practices encourage us to and I quote 

> Use the appropriate tags for each element (e.g., links, titles, etc.) and use HTML5 semantic tags (e.g., header, nav) over divs.

So the main tag, even though it's not being used at the moment, is a semantic tag so I will keep it on my code for now if that is okay with you.

Reply:
No problems in keeping the main tag. Good job searching for the best practices in HTML.